ow_lander/scripts/path_planning_commander.py

```diff
@@ -25,59 +25,81 @@
 import activity_deliver_sample
 import activity_grind
 
-# === MAIN COMMANDER CLASS =============================
-class MoveGroupPythonInteface(object):
+class PathPlanningCommander(object):
   
   def __init__(self):
-    super(MoveGroupPythonInteface, self).__init__()
+    super(PathPlanningCommander, self).__init__()
     moveit_commander.roscpp_initialize(sys.argv)
     robot = moveit_commander.RobotCommander()
     scene = moveit_commander.PlanningSceneInterface()
 
     move_arm = moveit_commander.MoveGroupCommander("arm")
-    move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    move_grinder = moveit_commander.MoveGroupCommander("grinder")
+    # move_limbs = moveit_commander.MoveGroupCommander("limbs")
+    # move_grinder = moveit_commander.MoveGroupCommander("grinder")
     display_trajectory_publisher = rospy.Publisher('/move_group/display_planned_path',
                                                    moveit_msgs.msg.DisplayTrajectory,
                                                    queue_size=20)
-    self.move_arm = move_arm
-    self.move_limbs = move_limbs
-    self.move_grinder = move_grinder
-
-# === SERVICE ACTIVITIES - guarded move =============================
-def handle_guarded_move(req): 
-  """
-  :type req: class 'ow_lander.srv._GuardedMove.GuardedMoveRequest'
-  """
-  try:
-    interface = MoveGroupPythonInteface()
+    self.arm_move_group = move_arm
+    # self.move_limbs = move_limbs
+    # self.move_grinder = move_grinder
+
+  # === SERVICE ACTIVITIES - Stow =============================
+  def handle_stow(self, req): 
+    """
+    :type req: class 'ow_lander.srv._Stow.StowRequest'
+    """
+    stowed_goal = self.arm_move_group.get_named_target_values("arm_stowed")
+    self.arm_move_group.go(stowed_goal, wait=True)
+    self.arm_move_group.stop()
+    return True, "Done"
+
+  # === SERVICE ACTIVITIES - Unstow =============================
+  def handle_unstow(self, req):
+    """
+    :type req: class 'ow_lander.srv._Unstow.UnstowRequest'
+    """
+    stowed_goal = self.arm_move_group.get_named_target_values("arm_unstowed")
```

Comment:
"stowed_goal" should be renamed "unstowed_goal". This unstowed configuration is different from the previous one.

Reply:
It uses the proper configuration, although the name of the variable is misleading. I will address that soon.

Reply:
Thanks. Still the arm configuration corresponding to "unstow" in this branch is different from the one in melodic-devel. Here the arm goes lower and the angles between links is not 90 degrees.

Reply:
joint values revised for arm_unstowed pose